standard/expressions.md

```diff
@@ -3494,6 +3752,15 @@ The term “correct grammar” above means only that the sequence of tokens shal
 <!-- markdownlint-enable MD028 -->
 > *Note*: From the disambiguation rule, it follows that, if `x` and `y` are identifiers, `(x)y`, `(x)(y)`, and `(x)(-y)` are *cast_expression*s, but `(x)-y` is not, even if `x` identifies a type. However, if `x` is a keyword that identifies a predefined type (such as `int`), then all four forms are *cast_expression*s (because such a keyword could not possibly be an expression by itself). *end note*
 
+If a cast expression `(T)E` invokes a user-defined conversion, then the null state (§Nullabilities-And-Null-States) of the expression is the default null state for the type of the user-defined conversion. Otherwise:
+
+- If `T` is a non-nullable value type then `T` has the null state “not null”
```

Comment:
Is something mixed up here? It is saying that `T` (a *type*), not the *expression*, has a null state.